frontend/src/metabase/query_builder/components/expressions/ExpressionEditorTextfield/helper-text-strings.ts

```diff
@@ -799,8 +805,28 @@ See the full list here: https://w.wiki/4Jx`,
   },
 ];
 
-export const getHelpText = (name: string): HelpText | undefined => {
-  return helperTextStrings.find(h => h.name === name);
+export const getHelpText = (
+  name: string,
+  database: Database,
+): HelpText | undefined => {
+  const helperText = helperTextStrings.find(h => h.name === name);
+
+  if (helperText && name === "now") {
```

Comment:
I think we should have 2 separate types in place of `HelpText`:
1. `HelpText` which is returned by this function with `description: string`
2. `HelpTextConfig` (or whatever) that has `description: () => string`

`getHelpText` would get the correct config item by name and execute its `description` callback to get up-to-date `description`. This way you won't need to add this checks here